v1.1/certificate-rotation.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Rotate Security Certificates
+summary: Rotate the security certificates of a secure CockroachDB cluster by generating and reloading new certificates.
+toc: false
+---
+
+<span class="version-tag">New in v1.1:</span> CockroachDB allows you to rotate security certificates without restarting the nodes. 
+
+<div id="toc"></div>
+
+## How CockroachDB Security Certificates Work
+
+A secure CockroachDB cluster uses [TLS](https://en.wikipedia.org/wiki/Transport_Layer_Security) for encrypted inter-node and client-node communication and requires CA, node, and client certificates and keys:
+
+File name pattern | File usage
+-------------|------------
+`ca.crt`     | CA certificate
+`node.crt`   | Server certificate
+`node.key`   | Key for server certificate
+`client.<user>.crt` | Client certificate for `<user>` (eg: `client.root.crt` for user `root`)
+`client.<user>.key` | Key for the client certificate
+
+The node and client certificates are signed by the CA certificate. 
+
+For each node, the CA certificate, node certificate, and node keys are uploaded to certs directory on the node. For each client, the CA certificate, client certificate, and client keys are uploaded to the client. 
+
+While establishing an inter-node or client-node communication, the nodes and clients use the CA certificate to verify each other's identity. 
+
+For more information, see [Create security certificates](create-security-certificates.html).
+
+## When to Rotate Certificates
+
+You may need to rotate the node, client, or CA certificates in the following scenarios:
+
+- The node, client, or CA certificates are expiring soon. 
+- Your organization's compliance policy may require periodical certificate rotation.
+- The key (for the node, client, or CA) is compromised.
+- You add a new load balancer and need to update the node certificates with the load balancer names and IP addresses. In this case, you need to rotate only the node certificates.
```

Comment:
A more general case might be better. eg: `You need to modify the contents of a certificate (eg: add another DNS name/ip address the node can be reached through, such as a load balancer)`